cmd/contour/serve.go

```diff
@@ -778,3 +835,57 @@ func informOnResource(clients *k8s.Clients, gvr schema.GroupVersionResource, han
 	inf.AddEventHandler(handler)
 	return nil
 }
+
+// commandOperatorRegexp parses the command operators used in Envoy access log configuration
```

Comment:
Was this moved in from the public API? Seems like it might be a good candidate to extract out to its own thing later.

Reply:
Yeah this is from parameters.go. This needs to live somewhere better than in serve.go, but needed a way to validate the accesslog fields.